README.md

```diff
@@ -192,11 +192,19 @@ lowercase. Make sure that the sentence type is defined in *src/nmea.h*.
 #include <time.h>
 #include <nmea.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
I'm not sure if the `extern "C"` is required for `struct` definitions as you can't have functions within a struct in C. However, if there are going to be any function declarations in this header file, this should definitely be kept.